configure.ac

```diff
@@ -326,7 +326,7 @@ echo
 
 # Dependencies
 AC_CHECK_OCAML_PKG([unix])
-AC_CHECK_OCAML_PKG([bigarray])
+AC_CHECK_OCAML_PKG([bigarray-compat])
```

Comment:
Shouldn't this allow both ?
```suggestion
AC_CHECK_OCAML_PKG([bigarray bigarray-compat])
```

Reply:
no, bigarray-compat only requires bigarray in OCaml < 4.07

Reply:
And `bigarray` does not exists at all in OCaml >= 5.00